api/v1alpha1/thinruntimeprofile_types.go

```diff
@@ -21,8 +21,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type NodePublishSecretPolicy string
+
+const (
+	NotMountNodePublishSecret        NodePublishSecretPolicy = "NotMountNodePublishSecret"
+	MountNodePublishSecretIfExists   NodePublishSecretPolicy = "MountNodePublishSecretIfExists"
+	CopyNodePublishSecretIfNotExists NodePublishSecretPolicy = "CopyNodePublishSecretIfNotExists"
```

Comment:
How about renaming `CopyNodePublishSecretIfNotExists` to `CopyToMountNodePublishSecretIfNotExists` ?

Reply:
Done. Renamed it with `CopyNodePublishSecretAndMountIfnotExists`